NetCasbin/InternalEnforcer.cs

```diff
@@ -44,7 +45,14 @@ protected bool InternalAddPolicy(string sec, string ptype, List<string> rule)
                 return false;
             }
 
-            NoticePolicyChanged(sec);
+            if (autoBuildRoleLinks && sec.Equals(PermConstants.Section.RoleSection))
```

Comment:
I checked the go version, the `autoBuildRoleLinks` condition shouldn't be here, because when group rules changed we always need to rebuild role links.

Reply:
You are right, we need to remove the `autoBuildRoleLinks` condition here.